app/models/student.rb

```diff
@@ -6,4 +6,8 @@ class Student < ApplicationRecord
   has_many :coordinators, through: :enrollments
 
   validates :first_name, presence: true, length: { minimum: 3 }
+
+  def name
```

Comment:
@nabeel- (only tagging since idk if you'll get a notification after this has been merged)

We discussed person inheritance earlier in the semester but decided not to do it since we wouldn't actually be abstracting any functionality; only properties. But now that we're sharing functionality, maybe?

*But* I'm still a fan of composition over inheritance in many situations. Maybe not this one, but as we move forward, I think it's worth considering. If someone can document all the times we need *person* stuff and abstract that out, I could see that working.

https://learnrubythehardway.org/book/ex44.html